### DISCUSSION TITLE
brew bump-cask-pr fails

### DISCUSSION THREAD

**Discussion by ChristianKraemer** [Homebrew/discussions#4374] [Everyday usage] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.0.10-117-g244359f
ORIGIN: https://github.com/Homebrew/brew
HEAD: 244359f314336754542873ad392d79999b107803
Last commit: 2 hours ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 873345b369f8744aa9524c157f0af5b57ddd0459
Core tap last commit: 20 minutes ago
Core tap branch: master
Core tap JSON: 31 Mar 17:31 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.NYh1Tyqqfw/org.xquartz:0
HOMEBREW_GITHUB_API_TOKEN: set
HOMEBREW_MAKE_JOBS: 4
HOMEBREW_NO_ENV_HINTS: set
HOMEBREW_NO_GOOGLE_ANALYTICS: set
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: quad-core 64-bit kabylake
Clang: 14.0.0 build 1400
Git: 2.40.0 => /usr/local/bin/git
Curl: 7.87.0 => /usr/bin/curl
macOS: 13.3-x86_64
CLT: 14.2.0.0.1.1668646533
Xcode: N/A
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

When running "brew bump-cask-pr --version 3.8.0  nextcloud" this fails with:

`Error: CLI arg for `sha265` is not declared for this command
Please report this issue:
  https://docs.brew.sh/Troubleshooting
/usr/local/Homebrew/Library/Homebrew/cli/args.rb:147:in `method_missing'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/bump-cask-pr.rb:87:in `bump_cask_pr'
/usr/local/Homebrew/Library/Homebrew/brew.rb:94:in `<main>'`

Anybody else experiencing problems on trying to bump casks?

**Comment by ZhongRuoyu** [comment, marked as answer]
That should be fixed in https://github.com/Homebrew/brew/pull/15116. Please do `brew update` and try again.

**Comment by ChristianKraemer** [comment]
Running again "brew update" fixed it. Before I wrote here I ran this 2 times as advised in the Troubleshooting. Maybe timing, CDN was not updated ...

**Reply by ZhongRuoyu** [reply]
It was not an issue on your side; the fix was merged less than one minute after you posted this. So this should be the fastest resolved discussion thread we've ever had!
